src/vm/i386/cgenx86.cpp

```diff
@@ -299,7 +299,12 @@ void TransitionFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 
     MethodDesc * pFunc = GetFunction();
     _ASSERTE(pFunc != NULL);
+
+#ifdef WIN64EXCEPTIONS
+    UpdateRegDisplayHelper(pRD, 0);
```

Comment:
@parjong It seems that the GetSP() result matches the Previous frame's sp is 0xffffa758 reported by gdb, so it looks correct. Does the pRD->pCurrentContext->Eip match the frame 13 (also reported as "saved eip" in the result of info frame 12?
And, how do these eip / esp values compare to the ones extracted using the former way?